charts/eap8/README.md

```diff
@@ -149,7 +149,7 @@ If the application image has been built by another mechanism, you can skip the b
 | `build.sourceSecret`|Name of the secret containing the credentials to login to Git source reposiory | - | The secret must exist in the same namespace or the chart will fail to install - [OKD documentation](https://docs.okd.io/latest/cicd/builds/creating-build-inputs.html#builds-manually-add-source-clone-secrets_creating-build-inputs) |
 | `build.triggers.genericSecret`| Name of the secret containing the WebHookSecretKey for the Generic Webhook | - | The secret must exist in the same namespace or the chart will fail to install - [OKD documentation](https://docs.okd.io/latest/cicd/builds/triggering-builds-build-hooks.html) |
 | `build.triggers.githubSecret`| Name of the secret containing the WebHookSecretKey for the GitHub Webhook | - | The secret must exist in the same namespace or the chart will fail to install - [OKD documentation](https://docs.okd.io/latest/cicd/builds/triggering-builds-build-hooks.html) |
-| `build.uri` | Git URI that references your git repo | &lt;required&gt; | Be sure to specify this to build the application. |
+| `build.uri` | (**required**) Git URI that references your Git repository | https://github.com/jboss-eap-up-and-running/eap8-getting-started | Be sure to specify this to build your own application. |
```

Comment:
Is it correct to call it `required` when default is provided when omitted?

Reply:
It is required from a schema perspective (https://github.com/jbossas/eap-charts/blob/51c29cbf6e9b06e853b61dd99e38f8c25315df09/charts/eap8/values.schema.json#L34).

The default value in `values.yaml` installs an example provided by us. In practice, an user must change that `uri` to point to their Git repo if they want to deploy their application.